src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -19,6 +23,25 @@ public void actionPerformed(ActionEvent e) {
 		switch (e.getActionCommand()) {
 			case "Figure Joe 1" -> FiguresJoe.figureScript1(driverManager.getCurrentDriver());
 			case "Figure Joe 2" -> FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+			case "My test" -> {
+				ComplexCommand complexCommand = new ComplexCommand();
+				complexCommand.addCommand(new SetPositionCommand(120, 120, driverManager.getCurrentDriver()));
+				complexCommand.addCommand(new OperateToCommand(-120, 120, driverManager.getCurrentDriver()));
+				complexCommand.addCommand(new OperateToCommand(-120, -120, driverManager.getCurrentDriver()));
+				complexCommand.addCommand(new OperateToCommand(120, -120, driverManager.getCurrentDriver()));
+				complexCommand.addCommand(new OperateToCommand(120, 120, driverManager.getCurrentDriver()));
+				complexCommand.execute();
+			}
+			case "Triangle Factory" -> {
+				FigureFactory factory = new FigureFactory(driverManager, 0, 0);
+				ComplexCommand triange = factory.createTriangle(120,0, 0, 120);
+				triange.execute();
+			}
+			case "Rectangle Factory" -> {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.